### PR TITLE
Composer fix

### PR DESCRIPTION
**Pull request by webda2l**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,15 +56,13 @@
     "config": {
         "sort-packages": true
     },
-    "extra": {
-        "branch-alias": {
```
Comment (jordisala1991) [inline composer.json]: Why do we need to remove branch alias?

Comment (webda2l) [reply composer.json]: Because https://github.com/a2lix/TranslationFormBundle/blob/master/composer.json#L62 should be the only one IMO.

Comment (jordisala1991) [reply composer.json]: But, for example: https://github.com/symfony/symfony/blob/3.4/composer.json

Symfony adds it on all branches.

Isnt it necessary to use the 2.x-dev alias?

Comment (webda2l) [reply composer.json]: I don't think so. It's true that Symfony and Api-Platform keep this alias in their composer but from what I understand, it has only usability for master branch.
And some people, remove it https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3555 in not master branch.

**Pull request merged**